### ISSUE TITLE
Support filtering function in `getResources`

### ISSUE THREAD

**Issue by jmeas** [jmeas/redux-resource#220]
I saw @marlonpp writing custom filtering logic in one app. It could be replaced with code like this:

```js
getResources(state, 'favorites', favorite => favorite.name.includes('pasta'));
```

Should we add that though? 🤔

**Comment by jmeas** [comment]
This seems related to #177 . Should we make this method full-featured? Or should we leave it minimally-useful, then rely on language features like:

```js
Object.values(state.books.resources).filter(r => r.name.includes('pasta'));
```

🤔

**Comment by jmeas** [comment]
Oh! What if you were passed the resource meta as the second argument. That'd be pretty cool.

```js
getResources(state, 'favorites', (favorite, favoriteMeta) => favorite.name.includes('pasta'));
```

**Comment by jmeas** [comment]
This has been released as `redux-resource@2.1.0`
